engine/src/test/java/io/camunda/zeebe/process/test/engine/db/InMemoryZeebeDbTransactionTest.java

```diff
@@ -512,10 +513,21 @@ void shouldNotIterateOverDeletionsInTransaction() throws Exception {
         });
   }
 
-  private enum ColumnFamilies {
-    DEFAULT, // rocksDB needs a default column family
-    ONE,
-    TWO,
-    THREE
+  private enum ColumnFamilies implements EnumValue {
```

Comment:
❓ Just a curiosity, why here we have an int value, and [here](https://github.com/camunda/zeebe/pull/16129/files#diff-f394f1f56b8a699fd006c23dd0a0b7519b05ae03b8097461def298c78350898fR480) instead we have the ordinal ?

Reply:
Because I was lazy [here](https://github.com/camunda/zeebe/pull/16129/files#diff-f394f1f56b8a699fd006c23dd0a0b7519b05ae03b8097461def298c78350898fR480). Static values are better 👍